### ISSUE TITLE
AMI 0.17 do not start on with i2.xlarge

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#901]
I use scylla 0.17 (ami-4431d024) successfully with m3.large for example, but with i2.xlarge it fail to start.
Its look like mkfs is stuck and cant progress.

This is what I see in journal:
```
Feb 11 09:22:21 ip-172-30-0-123.ec2.internal sudo[961]:   scylla : TTY=unknown ; PWD=/var/lib/scylla ; USER=root ; COMMAND=/usr/lib/scylla/scylla_prepare
Feb 11 09:22:58 ip-172-31-6-238 sudo[961]: Creating RAID0 for scylla using 1 disk(s): /dev/xvdb
```

```
$ df -k
Filesystem     1K-blocks    Used Available Use% Mounted on
devtmpfs        15697280       0  15697280   0% /dev
tmpfs           15707256       0  15707256   0% /dev/shm
tmpfs           15707256    8432  15698824   1% /run
tmpfs           15707256       0  15707256   0% /sys/fs/cgroup
/dev/xvda1      10474496 2456076   8018420  24% /
tmpfs            3141456       0   3141456   0% /run/user/1000
```

**Comment by avikivity** [comment]
mkfs issuse a BLKDISCARD operation which takes a long time.  Probably due to a newer kernel.  The BLKDISCARD operation makes sense,  and takes a long time because the disk is large, not because of a bug.


    16775 19:46:54.429627 ioctl(4, BLKDISCARD, [0, 5d22c400000]) = 0 <150.985063>

**Comment by tzach** [comment]
how much time can it take? do 10 min make sense?

**Comment by avikivity** [comment]
In the above log, it took two and a half minutes (on i2.8xlarge), but maybe it takes more depending on disk and instance conditions.

**Comment by slivne** [comment]
after 41 minutes on i2.xlarge it is still at it ....

**Comment by slivne** [comment]
after 1.5 hours - the mkfx.xfs is gone - yety it seems it did not complete

```
eb 14 20:45:48 ip-10-99-144-96 sudo[753]: Found disk /dev/xvdb
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: Creating RAID0 for scylla using 1 disk(s): /dev/xvdb
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: Loaded plugins: fastestmirror, langpacks
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: Loading mirror speeds from cached hostfile
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: * base: mirror.netdepot.com
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: * elrepo: mirror.symnds.com
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: * epel: mirror.symnds.com
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: * extras: mirror.cogentco.com
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: * updates: mirror.cogentco.com
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: Package mdadm-3.3.2-7.el7.x86_64 already installed and latest version
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: Package xfsprogs-3.2.2-2.el7.x86_64 already installed and latest version
Feb 14 20:45:48 ip-10-99-144-96 sudo[753]: Nothing to do
Feb 14 20:45:49 ip-10-99-144-96 sudo[753]: mdadm: Defaulting to version 1.2 metadata
Feb 14 20:45:49 ip-10-99-144-96 kernel: md: bind<xvdb>
Feb 14 20:45:49 ip-10-99-144-96 kernel: md: raid0 personality registered for level 0
Feb 14 20:45:49 ip-10-99-144-96 kernel: md/raid0:md0: md_size is 1562560000 sectors.
Feb 14 20:45:49 ip-10-99-144-96 kernel: md: RAID0 configuration for md0 - 1 zone
Feb 14 20:45:49 ip-10-99-144-96 kernel: md: zone0=[xvdb]
Feb 14 20:45:49 ip-10-99-144-96 kernel:       zone-offset=         0KB, device-offset=         0KB, size= 781280000KB
Feb 14 20:45:49 ip-10-99-144-96 kernel: 
Feb 14 20:45:49 ip-10-99-144-96 kernel: md0: detected capacity change from 0 to 800030720000
Feb 14 20:45:49 ip-10-99-144-96 sudo[753]: mdadm: array /dev/md0 started.
Feb 14 20:50:01 ip-10-99-144-96 systemd[1]: Created slice user-0.slice.
Feb 14 20:50:01 ip-10-99-144-96 systemd[1]: Starting user-0.slice.
Feb 14 20:50:01 ip-10-99-144-96 systemd[1]: Started Session 1 of user root.
Feb 14 20:50:01 ip-10-99-144-96 systemd[1]: Starting Session 1 of user root.
Feb 14 20:50:01 ip-10-99-144-96 CROND[2104]: (root) CMD (/usr/lib64/sa/sa1 1 1)
Feb 14 20:50:01 ip-10-99-144-96 systemd[1]: Removed slice user-0.slice.
Feb 14 20:50:01 ip-10-99-144-96 systemd[1]: Stopping user-0.slice.
Feb 14 20:58:54 ip-10-99-144-96 systemd[1]: Starting Cleanup of Temporary Directories...
Feb 14 20:58:54 ip-10-99-144-96 systemd-tmpfiles[2112]: Failed to open directory /var/lib/systemd/coredump: Too many levels of symbolic links
Feb 14 20:58:54 ip-10-99-144-96 systemd[1]: Started Cleanup of Temporary Directories.
Feb 14 20:59:57 ip-10-99-144-96 systemd[1]: scylla-server.service start-pre operation timed out. Terminating.
Feb 14 20:59:57 ip-10-99-144-96 systemd[1]: scylla-server.service: control process exited, code=exited status=143
Feb 14 20:59:57 ip-10-99-144-96 systemd[1]: Failed to start Scylla Server.
Feb 14 20:59:57 ip-10-99-144-96 systemd[1]: Dependency failed for Scylla JMX.
```

```
[centos@ip-10-99-144-96 ~]$ df -k
Filesystem     1K-blocks    Used Available Use% Mounted on
devtmpfs        15697280       0  15697280   0% /dev
tmpfs           15707256       0  15707256   0% /dev/shm
tmpfs           15707256    8464  15698792   1% /run
tmpfs           15707256       0  15707256   0% /sys/fs/cgroup
/dev/xvda1      10474496 2458536   8015960  24% /
tmpfs            3141456       0   3141456   0% /run/user/0
tmpfs            3141456       0   3141456   0% /run/user/1000
```

**Comment by avikivity** [comment]
systemd probably killed it.  We need to increase the timeout of scylla-ami-setup.service.

What does `systemctl status scylla-ami-setup` show?

**Comment by slivne** [comment]
the update to use a CentOS 7.2 base image was merged 3697cee76d15b57169fabd3336e1f63251348fe8

the update of the timeout is at #910

**Comment by slivne** [comment]
The update tto use CentOS 7.2 allows booting on i2.8x - closing this #910 still needs to be fixed
